AlphaWalletTests/Tokens/Coordinators/GetENSOwnerCoordinatorTests.swift

```diff
@@ -58,18 +64,21 @@ class GetENSOwnerCoordinatorTests: XCTestCase {
         expectations.append(expectation)
         let ensName = "1.offchainexample.eth"
         let server = makeServerForMainnet()
+        var fetchedAddress: AlphaWallet.Address?
+        var fetchedError: Error?
         firstly {
             ENSResolver(server: server).getENSAddressFromResolver(forName: ensName)
         }.done { address in
-            if address.sameContract(as: "41563129cdbbd0c5d3e1c86cf9563926b243834d") {
-                expectation.fulfill()
-            } else {
-                XCTFail("ENS name relying on ENSIP-10 did not resolve correctly")
-            }
-        }.catch { _ in
-            XCTFail("ENS name relying on ENSIP-10 did not resolve correctly")
+            fetchedAddress = address
+        }.ensure {
+            expectation.fulfill()
+        }.catch {error in
+            fetchedError = error
         }
         wait(for: expectations, timeout: 20)
+        XCTAssertNotNil(fetchedAddress)
```

Comment:
Move the asserts up according like https://github.com/AlphaWallet/alpha-wallet-ios/pull/4423#discussion_r858311795 ? Then merge when ready